docs/tutorials/embed/mvmds_tutorial.ipynb

```diff
@@ -37,12 +37,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Seems to be an error in this notebook. Not sure you wanted to add this notebook in the PR?